python_backend/api/company_survey.py

```
@@ -0,0 +1,55 @@
+from models.company import CompanyProfile
+
+router = APIRouter(
+    prefix="/surveys",
+    tags=["Company Survey"]
+)
+
+@router.post("/company/{company_id}")
+def submit_company_survey(
+    company_id: int,
+    survey_data: CompanySurvey,
+    profile_service: ProfileGeneratorService = Depends(),
```

Ruff (0.13.1)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)